dist/templates/components/podcast.html

```diff
@@ -0,0 +1,26 @@
+									<!-- PODCAST MODULE -->
+									<table>
+                                        <tr>
+                                            <td style="vertical-align: top; width: 40%;">
+                                                <div class="content" style="padding: 25px 0 0 20px;">
+                                                    <p><img src="https://media.sailthru.com/64d/1k7/7/6/64a70725dcba6.jpg" width="230" border="0" alt="Lorem ipsum." />
+													<div style="float:left;">June 6, 2023</div> <div style="float:right;">46:05 mins</div></p><br /><br />
+												<table width="100%" border="0" cellspacing="0" cellpadding="0" style="border-radius: 0 0 5px 5px; padding-top:15px;">
```

Comment:
table is missing `role="presentation"`

If we are not using a table for it intended purpose, we need to add that to all table for screen readers. Otherwise it will read cell 1 {content}, cell 2 {content}, and so on